modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/js/form_builder_render_rule_condition.js

```diff
@@ -513,7 +513,9 @@ AUI.add(
 				var value = [];
 
 				if (condition) {
```

Comment:
Instead of two nested ifs, we can write: if (condition && condition.operator != '').

Reply:
outdated code